lib/rubocop/cop/performance/big_decimal_with_numeric_argument.rb

```diff
@@ -10,36 +10,47 @@ module Performance
       # @example
       #   # bad
       #   BigDecimal(1, 2)
+      #   1.to_d(2)
       #   BigDecimal(1.2, 3, exception: true)
+      #   1.2.to_d(3, exception: true)
       #
       #   # good
       #   BigDecimal('1', 2)
+      #   BigDecimal('1', 2)
+      #   BigDecimal('1.2', 3, exception: true)
```

Comment:
My understanding was bad. These are duplicated example code. Can you remove it if you like? 💦

Reply:
That only goes to show that I chose badly with the examples, so I changed it to use different digits to make it obvious.